doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Authoring/DataSets/data-sets-authoring-con.adoc

```diff
@@ -0,0 +1,10 @@
+[id='data-sets-authoring-con_{context}']
+= Data sets authoring
+
+A data set is a collection of related sets of information. It can be stored in many ways, such as in a database, in a Microsoft Excel file, or in memory. A data set definition instructs {CENTRAL} methods to access, read, and parse a data set. {CENTRAL} does not store data. It enables you to define access to a data set regardless of where the data is stored.
+
+For example, if data is stored in a database, a valid data set could contain the entire database or a subset of the database as a result of an SQL query. In both cases, the data is used as input for the reporting components of {CENTRAL} which then displays the information.
+
+To access a data set, you must create and register a data set definition. The data set definition specifies the location of the data set, the ways to access it, read it, and parse it, and the columns that it contains.
```

Comment:
Minor rewrite suggestion with a question. For "and the columns that it contains." Do you mean the column names? If so, I would say "and the definition column names.

The data set definition specifies the location of the data set, options to access it, read it, parse it, and the columns that it contains.

Reply:
Done.